deepmd/common.py

```diff
@@ -483,3 +485,39 @@ def get_np_precision(precision: "_PRECISION") -> np.dtype:
         return np.float64
     else:
         raise RuntimeError(f"{precision} is not a valid precision")
+
+def load_model_info(model_file: str) -> Tuple[dict, float]:
+    """Get numpy precision constant from string.
+
+    Parameters
+    ----------
+    model_file : str
+        The input frozen model, which will be compressed by the deepmd-kit.
+
+    Returns
+    -------
+    jdata
+        The training script saved in the frozen model
+    min_nbor_dist
+        The nearest distance between neighbor atoms saved in the frozen model
+
+    Raises
+    ------
+    GraphWithoutJdataOrMinNborDistError
+        If the training script or min_nbor_dist are within the frozen model
+    """
+    graph_def = tf.GraphDef()
+    with open(model_file, "rb") as f:
+        graph_def.ParseFromString(f.read())
+    with tf.Graph().as_default() as graph:
+        tf.import_graph_def(graph_def, name = "")
+        try:
+            jdata = graph.get_tensor_by_name('training_script:0')
```

Comment:
Shall we collect these constant strings into an unified Python file like `constant.py`? Then:
- Name consistency between saver and reader can be ensured.
- Other guys may reference them in individual Python programs.

Reply:
good idea

Reply:
I see